tests/detect-dce-opnum/test.yaml

```diff
@@ -0,0 +1,21 @@
+args:
+- -k none
+
+checks:
+- filter:
+    count: 2
+    match:
+      event_type: dcerpc
```

Comment:
Logging was not introduced till 6.0.x for dcerpc so there are no events for that. Please add a min-version check for 6 here.